### DISCUSSION TITLE
Support of Android Managment API

### DISCUSSION THREAD

**Discussion by DanielKenREK** [rustdesk/rustdesk#3892] [Ideas]
Do you support the android management api? 

It would be helpful if the host and the key can be provided in a enterprise policy by json.

**Comment by prasanth1308** [comment]
Hi @DanielKenREK , Are meaning to  implement Managed configuration  for the app?

**Reply by CynthiaArmstrong** [reply]
I suppose that he would love the same feature like @shaqaruden in https://github.com/rustdesk/rustdesk/discussions/4332.
